models/role.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Role = sequelize.define(
+    'role',
```

Comment:
should be `roles`
- we defined the table name to be `roles` in the migration
- table names should be plural as they store multiple entries of a single type of entity
- the model/export name (i.e. `const role = ...; ... export role;`) should be singular as we generally work with a single instance of an entity/model in code

hopefully that helps clear it up some
```suggestion
    'roles',
```